teamcity/messages.py

```diff
@@ -10,6 +10,14 @@
     text_type = str
 
 
+quote = {"'": "|'", "|": "||", "\n": "|n", "\r": "|r", ']': '|]'}
```

Comment:
Went ahead and added the `[` => `|[` mapping in 12a60c6. I figure that @shalupov can cherry-pick if he wants some commits but not others.